### DISCUSSION TITLE
Devo criar os projetos já com uma solução?

### DISCUSSION THREAD

**Discussion by victor-sc-92** [balta-io/forum#264] [Tira Dúvidas] [answered]
Aula: [.Net/Tipos de projeto](https://balta.io/player/assistir/5ef614da-366d-6156-d049-694800000000/0482ee81-8f83-4653-a421-000028010404)

Tem tutoriais que as pessoa sempre cria uma solução para o projeto, já na aula se cria o projeto sem uma solução, mas em um projeto real é comum criar sem solução mesmo? ou devemos sempre criar com uma solução mesmo que tenha apenas 1 projeto?

Exemplo do tutorial que vi:
```sh
$ dotnet new sln -n <nome_solução>
$ cd <nome_solução>
$ dotnet new globaljson --sdk-version <versão>
$ dotnet new <mvc | webapi | web> -o <nome_projeto> [--auth <Individual>] [-f <versão_net>]
$ dotnet sln add ./<nome_projeto>/<nome_projeto>.csproj
$ cd <nome_projeto>
```

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @CodeAkio, como vai?

Lá na frente, nos cursos de ASP.NET, criaremos soluções e entenderemos para que elas funcionam. No momento, só criar os projetos basta.
